configure.ac

```diff
@@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
 # For host type checks
 AC_CANONICAL_HOST
 
+#For a working getsockopt() optname=SO_ERROR
+dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
```

Comment:
On the Debian bug discussion [1], @sthibaul mentioned that this sockopt is in fact available on Hurd. Also doesn't look like HAVE_SOCKLEN_T is used. Any reason to keep this section?

[1] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=799235